Doc/whatsnew/3.8.rst

```diff
@@ -234,6 +234,14 @@ Added method :meth:`~tkinter.Canvas.moveto`
 in the :class:`tkinter.Canvas` class.
 (Contributed by Juliette Monsel in :issue:`23831`.)
 
+time
+-------
+
+Added a new clock identifier `~time.CLOCK_UPTIME_RAW` for Darwin that increments
```

Comment:
Darwin is correct, but I prefer "macOS" since this name is more commonly used than Darwin (in my experience).